docs/hld/projects/publish/brief.md

```diff
@@ -0,0 +1,14 @@
+# Publishing Marxan Cloud projects
+This document describes the current high-level overview of the Marxan Cloud platform user features related to publishing of projects. It is a working document, meant to gather feedback on the requirements before moving on to high level design and detailed implementation plan.
+
+## Aim
+Users should be able to publish Marxan Cloud projects to the community for a range of purposes:
+- A user may share a project with people outside of his organization.
+- Anyone can browse all published projects.
+
+# User actions
+According to the [design](https://www.figma.com/file/hq0BZNB9fzyFSbEUgQIHdK/Marxan-Visual_V02?node-id=4671%3A8116) project publishing is available under a dedicated button.
+
+This moves us to the [configuration](https://www.figma.com/file/hq0BZNB9fzyFSbEUgQIHdK/Marxan-Visual_V02?node-id=4684%3A7130).
+
+After choosing all new information the project can be published.
```

Comment:
I think this should be in a new HLD

Reply:
I will put it in a new one. I trust you that it will be simply HLD, but just make it easy to find I prefer to put it into a dedicated one. I want to prevent that someone would think that this is not documented when it will be not presented in the docs easly.